hardhat.config.ts

```diff
@@ -0,0 +1,15 @@
+import { HardhatUserConfig } from 'hardhat/types';
+
+const config: HardhatUserConfig = {
+  defaultNetwork: 'localhost',
```

Comment:
As an alternative you could also add `--network localhost` to the `eth-node` script to avoid changing the defaultNetwork and you might even not need to set anything but the compiler version in here.